src/ResourceManager/AzureBatch/Commands.Batch/Models/ListNodeFileOptions.cs

```diff
@@ -76,9 +76,9 @@ public class ListNodeFileOptions : BatchClientParametersBase
         public PSComputeNode ComputeNode { get; private set; }
 
         /// <summary>
-        /// If specified, the single node file with this name will be returned.
+        /// If specified, the single node file with this path will be returned.
         /// </summary>
-        public string NodeFileName { get; set; }
+        public string Path { get; set; }
```

Comment:
should we keep the original property since it isn't obsoleted?

Reply:
My intent is to just leave this as is (no obsolete) since this is a breaking change, unless otherwise suggested by one of the powershell maintainers.

Reply:
The change in usage needs to be documented in upcoming-breaking-changes

Reply:
This is actually only a type used internally in the cmdlets, it's not part of a cmdlet or return type. It's just used to store information about what request is being performed so we can pass it off to the C# SDK, so I don't think there's anything to really say in upcoming-breaking-changes. It was already covered in the `Name` -> `Path` change for the `GetNodeFile` cmdlet.